coderd/provisionerdserver/provisionerdserver.go

```
@@ -125,6 +127,9 @@ func NewServer(
 	options Options,
 ) (proto.DRPCProvisionerDaemonServer, error) {
 	// Panic early if pointers are nil
```

(not strictly related with this PR)
nit: panic might not be relevant, it is rather fail fast

---

That comment is left over from an earlier draft of this code where we panic'd instead of returning an error.